spotbugs/etc/findbugsfilter.xsd

```diff
@@ -264,137 +317,151 @@
         </annotation>
         <attribute name="value" type="fb:PriorityValueType">
             <annotation>
-                <documentation>1: high-priority warnings, 2: medium-priority warnings, 3: low-priority warnings</documentation>
-            </annotation></attribute>
+                <documentation>1: high-priority warnings, 2: medium-priority warnings, 3: low-priority warnings
+                </documentation>
+            </annotation>
+        </attribute>
     </complexType>
 
     <complexType name="ConfidenceType">
         <annotation>
-            <documentation></documentation>
+            <documentation/>
         </annotation>
         <attribute name="value" type="fb:ConfidenceValueType">
             <annotation>
-                <documentation>1: high-priority warnings, 2: medium-priority warnings, 3: low-priority warnings</documentation>
-            </annotation></attribute>
+                <documentation>1: high-priority warnings, 2: medium-priority warnings, 3: low-priority warnings
+                </documentation>
+            </annotation>
+        </attribute>
     </complexType>
 
     <complexType name="RankType">
         <annotation>
-            <documentation></documentation>
+            <documentation/>
         </annotation>
         <attribute name="value" type="fb:RankValueType">
             <annotation>
-                <documentation>1 to 4 are scariest, 5 to 9 scary, 10 to 14 troubling, and 15 to 20 of concern bugs.</documentation>
-            </annotation></attribute>
+                <documentation>1 to 4 are scariest, 5 to 9 scary, 10 to 14 troubling, and 15 to 20 of concern bugs.
+                </documentation>
+            </annotation>
+        </attribute>
     </complexType>
 
     <simpleType name="PriorityValueType">
         <restriction base="int">
-            <minInclusive value="1"></minInclusive>
-            <maxInclusive value="3"></maxInclusive>
+            <minInclusive value="1"/>
+            <maxInclusive value="3"/>
         </restriction>
     </simpleType>
 
     <simpleType name="ConfidenceValueType">
         <restriction base="int">
-            <minInclusive value="1"></minInclusive>
-            <maxInclusive value="3"></maxInclusive>
+            <minInclusive value="1"/>
+            <maxInclusive value="3"/>
         </restriction>
     </simpleType>
 
     <simpleType name="RankValueType">
         <restriction base="int">
-            <minInclusive value="1"></minInclusive>
-            <maxInclusive value="20"></maxInclusive>
+            <minInclusive value="1"/>
+            <maxInclusive value="20"/>
         </restriction>
     </simpleType>
 
     <complexType name="MatchType">
-        <sequence maxOccurs="unbounded" minOccurs="0">
```

Comment:
Why not simply switch from `minOccurs="0"` to `minOccurs="1"` here? Why making the bigger change to the nested `choice` element?